packages/jest-runtime/src/script_transformer.js

```diff
@@ -517,11 +520,25 @@ const calcIgnorePatternRegexp = (config: ProjectConfig): ?RegExp => {
   return new RegExp(config.transformIgnorePatterns.join('|'));
 };
 
-const wrap = content =>
-  '({"' +
-  ScriptTransformer.EVAL_RESULT_VARIABLE +
-  '":function(module,exports,require,__dirname,__filename,global,jest){' +
-  content +
-  '\n}});';
+const wrap = (content, ...extras) => {
```

Comment:
can we add some unit tests that calls `transform` an verifies the wrapped code looks correct?